relentless/simulate/lammps.py

```diff
@@ -0,0 +1,347 @@
+import abc
+import os
+
+import numpy as np
+
+from relentless.core.volume import TriclinicBox
+from . import simulate
+
+try:
+    import lammps
+    _lammps_found = True
+except ImportError:
+    _lammps_found = False
+
+class LAMMPS(simulate.Simulation):
+    def __init__(self, operations, quiet=True, **options):
+        if not _lammps_found:
+            raise ImportError('LAMMPS not found.')
+        elif lammps.lammps().version() < 20201029:
+            raise ImportError('Only LAMMPS 29 Oct 2020 or newer is supported.')
```

Comment:
Ah, I forgot that `version()` is only a property of a _running_ lammps instance. We don't want to make one of these before we actually need it just to check the version, so move this check down into `_new_instance` (after you create `sim.lammps`).